pkg/daemon/file_writers.go

```diff
@@ -45,12 +46,50 @@ func noOrigFileStampName(fpath string) string {
 	return filepath.Join(noOrigParentDir(), fpath+".mcdnoorig")
 }
 
-func createOrigFile(fromPath, fpath string) error {
-	if _, err := os.Stat(noOrigFileStampName(fpath)); err == nil {
-		// we already created the no orig file for this default file
-		return nil
+func createOrigFile(fromPath, fpath string, isCoreOSVariant, forceCreate bool) error {
+
+	// Declare a boolean variable - orig - to indicate whether an orig file should be created
```

Comment:
nit: I think this set of comments can be skipped, since the code does a good job explaining what this bool is used for